src/webhooks/handleDisputeLost.ts

```diff
@@ -0,0 +1,52 @@
+import Stripe from 'stripe';
+import { UsersService } from '../services/users.service';
+import { StorageService } from '../services/storage.service';
+import CacheService from '../services/cache.service';
+import { AppConfig } from '../config';
+import handleLifetimeRefunded from './handleLifetimeRefunded';
+import { PaymentService } from '../services/payment.service';
+import { FastifyBaseLogger } from 'fastify';
+
+interface HandleDisputeLostProps {
+  charge: Stripe.Dispute;
+  stripe: Stripe;
+  paymentService: PaymentService;
+  usersService: UsersService;
+  storageService: StorageService;
+  cacheService: CacheService;
+  log: FastifyBaseLogger;
+  config: AppConfig;
+}
+
+export async function handleDisputeLost({
```

Comment:
`handleDisputeResult` could be a better name, as you do not know if the dispute has been lost or not, for what I see in the below checks